db/installationInit.sql

```diff
@@ -171,12 +213,33 @@ INSERT INTO `SelectConfiguration_OPTIONS` (`SelectConfiguration_id`, `options`)
 (1, 'en_GB'),
 (3, 'number'),
 (3, 'text'),
-(33, 'de.imi.mopat.helper.controller.RandomPatientDataRetrieverImpl'),
-(33, 'de.imi.mopat.helper.controller.HL7v22PatientInformationRetriever'),
-(33, 'de.imi.mopat.helper.controller.DummyPatientDataRetrieverImpl'),
+(81, 'de.imi.mopat.helper.controller.RandomPatientDataRetrieverImpl'),
+(81, 'de.imi.mopat.helper.controller.HL7v22PatientInformationRetriever'),
+(81, 'de.imi.mopat.helper.controller.DummyPatientDataRetrieverImpl'),
+(81, 'de.imi.mopat.helper.controller.HL7v22PatientInformationRetrieverByPID'),
 (38, 'de_DE'),
 (38, 'en_GB');
 
+CREATE TABLE IF NOT EXISTS `SelectClinicConfiguration_OPTIONS` (
```

Comment:
This table is still created, but it seems like it is not really used anymore, as we moved away from configuring this in the clinic view directy. Should be deleted then